### ISSUE TITLE
ERROR: float() argument must be a string or a number, not '_ImageCrop'

### ISSUE THREAD

**Issue by endpang** [DmitryUlyanov/deep-image-prior#9]
get_image  yet return  pil_to_np result。

modify code

> img_np = get_image(fname, imsize)[1]

run success。

**Comment by tuanphuc** [comment]
Update to the latest version of pillow solves the error.
`pip3 install pillow --upgrade`

**Comment by ThGravo** [comment]
No, it doesn't... ;) I'm using pillow 5.0.0 and get this error.
